session/Session.tsx

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const BASE_URL = 'https://wu.up.krakow.pl/WU/';
+
+export const LOGIN_URL = 'Logowanie2.aspx';
+export const STUDIES_FIELDS_URL = 'KierunkiStudiow.aspx';
+export const PROFILE_URL = 'Wynik2.aspx';
+export const GRADES_URL = 'OcenyP.aspx';
+export const SCHEDULE_URL = 'PodzGodzin.aspx';
+
+export const axiosInstance = axios.create({
+  baseURL: BASE_URL,
+  headers: {'Content-Type': 'application/x-www-form-urlencoded'},
```

Comment:
Remove headers from global session, not every request needs to be type application/x-www-form-urlencoded

Reply:
While using, you can overwrite the header section. Example
`const response = await axiosInstance.post(
          LOGIN_URL,
          {data: loginData},
          {
            headers: {'Content-Type': 'THIS'},
          },
`